src/cozmo/run.py

```diff
@@ -847,4 +848,8 @@ def wrapper(sdk_conn):
     except KeyboardInterrupt:
         logger.info('Exit requested by user')
     except exceptions.ConnectionError as e:
-        sys.exit("A connection error occurred: %s" % e)
+        if throw_error:
+            logger.error("A connection error occurred: %s" % e)
+            raise e
```

Comment:
Also note, as per our coding standard:
"To re-raise the original exception use raise by itself or raise MyException() from exc (rather than raise exc) to maintain the original stack trace."
(so this should probably just be `raise`)

Reply:
Okey Dokey :)